graylog2-server/src/test/java/org/graylog2/indexer/indices/IndicesIT.java

```diff
@@ -126,16 +119,28 @@ public void testDelete() {
     public void testClose() {
         final String index = client().createRandomIndex("indices_it_");
 
-        assertThat(indicesAdapter.isOpen(index)).isTrue();
+        assertThat(indices.isOpen(index)).isTrue();
 
         indices.close(index);
 
-        assertThat(indicesAdapter.isClosed(index)).isTrue();
-        //assertThat(getIndexState(index)).isEqualTo("close");
+        assertThat(indices.isClosed(index)).isTrue();
     }
 
     @Test
-    public void testAliasExists() {
+    public void findClosedIndices() {
+        final String index1 = client().createRandomIndex("indices_it_");
+        client().closeIndex(index1);
+        final String index2 = client().createRandomIndex("otherindices_it_");
+        client().closeIndex(index2);
+        final String index3 = client().createRandomIndex("evenmoreindices_it_");
```

Comment:
The variable for `index3` is not needed. If you want to keep it in here to make it more obvious that it shouldn't be contained in the result, the "unused" warning should be suppressed.

Reply:
:heavy_check_mark: